lmcache/address_manager/disk_address_manager.py

```diff
@@ -0,0 +1,285 @@
+import os
+import re
+import threading
+from collections import OrderedDict
+from dataclasses import dataclass
+from typing import Iterable, Optional
+from xmlrpc.server import SimpleXMLRPCRequestHandler, SimpleXMLRPCServer
+
+import yaml
+
+from lmcache.logging import init_logger
+from lmcache.storage_backend.evictor import LRUEvictor
+from lmcache.storage_backend.evictor.base_evictor import PutStatus
+from lmcache.utils import CacheEngineKey, DiskCacheMetadata
+
+logger = init_logger(__name__)
+
+
+@dataclass
+class LMCAddressManagerConfig:
```

Comment:
Instead of having a new configuration class defined here, can we reuse the `LMCacheEngineConfig` (i.e., pass `LMCacheEngineConfig` to `LMCDiskAddressManager.__init__()`?

This is because if someone wants to add other ways to initialize the configuration (for instance, initialize from environment variables), he/she is likely to only look at the LMCacheEngineConfig and may not know this class needs to be updated as well.